policyengine_us/variables/gov/irs/tax/payroll/medicare/employer_medicare_tax.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
```

Comment:
@MaxGhenis should we add the employer side taxes to the net income tree as they are?